zkevm-circuits/src/evm_circuit/execution/mulmod.rs

```diff
@@ -206,14 +206,14 @@ mod test {
         test(a.into(), b.into(), n.into(), r.map(Word::from), true)
     }
 
-    fn test_ko_u32(a: u32, b: u32, n: u32, r: Option<u32>) {
-        test(a.into(), b.into(), n.into(), r.map(Word::from), false)
-    }
+    // fn test_ko_u32(a: u32, b: u32, n: u32, r: Option<u32>) {
+    //     test(a.into(), b.into(), n.into(), r.map(Word::from), false)
+    // }
 
     #[test]
     fn mulmod_simple() {
         test_ok_u32(7, 12, 10, None);
-        test_ok_u32(7, 1, 10, None);
+        // test_ok_u32(7, 1, 10, None);
```

Comment:
forgot to enable  this test ?

Reply:
cuz stack reconstruct always produce correct output, negative test here not works anymore

Reply:
adversary can make wrong output by modifying code in any way. if u don't want to do it now, at least add `TODO` item in the comment at the moment

Reply:
@lightsing add TODO